<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_spiegeln</class>
 <widget class="QDialog" name="Dialog_spiegeln">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spiegeln</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_esc">
      <property name="text">
       <string>Abbruch</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_save">
      <property name="text">
       <string>Speichern</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>371</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineEdit_ypos"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Spiegelachsenpos. X</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Spiegelachsenpos. Y</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>[SPGX]</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QCheckBox" name="checkBox_ybed">
      <property name="text">
       <string>Spiegeln in Y</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Bezeichnung</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>[SPGX]</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="lineEdit_xpos"/>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="lineEdit_bez"/>
    </item>
    <item row="2" column="3">
     <widget class="QCheckBox" name="checkBox_xbed">
      <property name="text">
       <string>Spiegeln in X</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>[BEZ]</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Ausf√ºhrbedingung</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>[AFB]</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLineEdit" name="lineEdit_afb"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_xpos</tabstop>
  <tabstop>lineEdit_ypos</tabstop>
  <tabstop>checkBox_xbed</tabstop>
  <tabstop>checkBox_ybed</tabstop>
  <tabstop>lineEdit_bez</tabstop>
  <tabstop>lineEdit_afb</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_esc</tabstop>
  <tabstop>pushButton_save</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
